src/Legacy/Storage.php

```diff
@@ -2557,7 +2558,8 @@ protected function getRelation($content)
         }
 
         // Get the contenttype from first $content
-        $contenttype = $content[util::array_first_key($content)]->contenttype['slug'];
+        $ct = $content[util::array_first_key($content)];
```

Comment:
This has changed in 3.3, and this will nuke that … https://github.com/bolt/bolt/blob/release/3.3/src/Legacy/Storage.php#L2383-L2384